forge/routes/auth/index.js

```diff
@@ -44,18 +44,16 @@ module.exports = fp(async function (app, opts, done) {
                 if (scheme !== 'Bearer') {
                     reply.code(401).send({ code: 'unauthorized', error: 'unauthorized' })
                 }
-                if (/^ff[td]/.test(token)) {
```

Comment:
What is this going to do to live sessions at the point of upgrade?

Are we going to log all existing sessions out?

Reply:
A side-effect will be to invalidate existing edit sessions. We *might* be able to add a migration that moves the appropriate tokens from one table to the other.... will take a look.